apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -386,34 +421,41 @@ void CameraPipe::generate() {
         } else if (get_target().has_feature(Target::HVX_128)) {
             vec = 64;
         }
+        processed.compute_root()
+            .reorder(c, x, y)
+            .split(y, yi, yii, 2, TailStrategy::RoundUp)
+            .split(yi, yo, yi, strip_size / 2)
+            .vectorize(x, 2*vec, TailStrategy::RoundUp)
+            .unroll(c)
+            .parallel(yo);
+
         denoised.compute_at(processed, yi).store_at(processed, yo)
             .prefetch(input, y, 2)
-            .fold_storage(y, 8)
+            .fold_storage(y, 16)
```

Comment:
What are the fold_storage constants based on? Is it worthy pulling them into named constants for clarity?

Reply:
Honestly they're based on trial and error. Stencils combine to produce support requirements that are hard to understand. I'm not sure that named constants help here. When reading the schedule, I feel like the value of the constant matters a lot more than the concept communicated by the name of a constant in these use cases.